packages/uui-textarea/lib/uui-textarea.element.ts

```diff
@@ -305,8 +226,9 @@ export class UUITextareaElement extends FormControlMixin(LitElement) {
 
     input.style.height = 'auto';
 
-    if (input.scrollHeight > input.clientHeight) {
-      input.style.height = input.scrollHeight + 'px';
+    // Note: Add + 2 because of the border top+bottom 1px each
```

Comment:
Since we don't measure the border thickness here, we should make sure to mention that there is a hardcoded 2px in the autoUpdateHeight as a comment next to the style where the border is set.